spec/models/tattoo_spec.rb

```diff
@@ -3,7 +3,8 @@
 RSpec.describe Tattoo, type: :model do
 
   context 'tattoos' do
-    let!(:tattoo) { FactoryGirl.create(:tattoo) }
+    let!(:tattoo) { FactoryGirl.create(:tattoo,
+    title: 'Badass Celtic Armband') }
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Expression at 7, 37 should be on its own line.